# Define the goal
geometry_msgs/Twist twist
duration target_duration
---
# Define the result
bool success
string error_message
geometry_msgs/Pose[] poses
time[] time
---
# Define a feedback message
string message
